src/Stripe.net/Services/WebhookEndpoints/WebhookEndpointCreateOptions.cs

```diff
@@ -7,6 +7,17 @@ namespace Stripe
 
     public class WebhookEndpointCreateOptions : WebhookEndpointSharedOptions
     {
+        /// <summary>
+        /// Events sent to this endpoint will be generated with this API version instead of your
+        /// account's default API version. We recommend that you set this to the API version that
+        /// the library is pinned to in <code>StripeConfiguration.stripeApiVersion</code>
+        /// </summary>
+        [JsonProperty("api_version")]
```

Comment:
Thank you for the comments! 
Given that in .NET we have this check which only gets triggered during real integration maybe we should add a comment here: "If set with different from this pinned version, 
set the `throwOnApiVersionMismatch` to false in `<code>EventUtility.ParseEvent</code>`
https://github.com/stripe/stripe-dotnet/blob/4b316d90ce0e481362a65969709b6a1e52e07faf/src/Stripe.net/Services/Events/EventUtility.cs#L21-L31